### PR TITLE
UX updates

### PR DESCRIPTION
**Pull request by evanyang1**
Update 1: You can now press 'Enter' instead of clicking the 'Log In' button. This improves my own user experience. 

To do: 
Implement confirm password feature

### CODE REVIEW

**Comment by evanyang1** [comment]
Confirm pw is in place

File: client/src/pages/Register.js [hunk 1]
```diff
@@ -59,43 +59,53 @@ export default function Register() {
...
+  const handleSubmit = async () => {
     if (!isEmail(email)) setErrorEmail(true);
-    if (!isName(name)) setErrorName(true);
+    //if (!isName(name)) setErrorName(true);
+    if (!name) setErrorName(true);
     if (!password) setErrorPassword(true);
     if (!primaryLanguage) setErrorLanguage(true);
+    if (password !== confirmPassword) {
+      setErrorConfirmPassword(true);
+    }
```
Comment (ThinkTink) [inline client/src/pages/Register.js:94]: Can we move all this into isErrorFree? We only want to check each thing once.
Also, we don't need it async unless we plan on awaiting something

File: client/src/pages/Register.js [hunk 2]
```diff
@@ -59,43 +59,53 @@ export default function Register() {
...
 
   const [errorName, setErrorName] = useState(false);
   const [errorEmail, setErrorEmail] = useState(false);
   const [errorPassword, setErrorPassword] = useState(false);
+  const [errorConfirmPassword, setErrorConfirmPassword] = useState(false);
   const [errorLanguage, setErrorLanguage] = useState(false);
 
   const isEmail = (email) => /^\S+@\S+$/.test(email);
-  const isName = (name) => /^[A-Z]+$/i.test(name);
+  //const isName = (name) => /^[A-Z]+$/i.test(name);
```
Comment (ThinkTink) [inline client/src/pages/Register.js:72]: Are we getting rid of this? Let's either uncomment it or completely delete it

File: client/src/pages/Register.js [hunk 3]
```diff
@@ -59,43 +59,53 @@ export default function Register() {
...
   useEffect(() => {
-    if (isName(name)) setErrorName(false);
+    //if (isName(name)) setErrorName(false);
+    if (name) setErrorName(true);
     if (isEmail(email)) setErrorEmail(false);
     if (password.length > 5) setErrorPassword(false);
     if (primaryLanguage) setErrorLanguage(false);
-  }, [name, email, password.length, primaryLanguage]);
+    if (password === confirmPassword) setErrorConfirmPassword(false);
+  }, [name, email, password.length, primaryLanguage, confirmPassword]);
```
Comment (ThinkTink) [inline client/src/pages/Register.js:108]:
```suggestion
  }, [name, email, password, primaryLanguage, confirmPassword]);
```

**Pull request merged**